presto-docs/src/main/sphinx/release/release-0.275.rst

```diff
@@ -0,0 +1,36 @@
+=============
+Release 0.275
+=============
+
+**Highlights**
+==============
+
+**Details**
+===========
+
+General Changes
+_______________
+* Fix Disaggregated Coordinator to correctly identify running queries as leaked.
+* Improve the memory limit error message with allocationTag.
+* Add :func:`trimmed_mean` for :ref:`tdigest <tdigest_type>`. The prototype of the function is: trimmed_mean(tdigest: TDIGEST, lower_quantile: DOUBLE, upper_quantile: DOUBLE) -> DOUBLE.
+* Add a router in front of presto clusters. The router is deployed as a standalone service with a presto-styled UI.
+* Add new fields in Iceberg files system table.
+* Add raft server to resource manager with disaggregated coordinators using Apache Ratis. Raft server enabled in resource manager using the ``isEnabled`` property in Config file.
+* Add session property ``query_max_output_positions`` and configuration property ``query.max-output-positions`` to control how many rows a query can output. The query might end up returning more rows than the limit as the check is asynchronous.
+* Add support for setting session property for presto on spark.
+* Add plan canonicalization strategies to be used by HBO framework
+* Add `CONSTRUCT_TDIGEST()` for `Tdigest`. The prototype of the function is: CONSTRUCT_TDIGEST( double[] centroidMeans, double[] centroidWeights, double compression, double min, double max, double sum, bigint count) -> Slice
```

Comment:
Same comments as above. Also, if CONSTRUCT_TDIGEST is a function, we should use ":func:`trimmed_mean`"